### PR TITLE
Split plugin.go into several files

### PR DESCRIPTION
**Pull request by vdemeester**
This is *just a nit*, but to be consistent with other "objects" 🐮.
Unit testing will come later 👼.

/cc @tiborvass @thaJeztah @runcom @calavera 

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, should we wait for @tiborvass?

File: client/plugin_install.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	if err := json.NewDecoder(resp.body).Decode(&privileges); err != nil {
+		return err
+	}
+	ensureReaderClosed(resp)
+
+	if !acceptAllPermissions && len(privileges) > 0 {
+
+		fmt.Fprintf(out, "Plugin %q requested the following privileges:\n", name)
+		for _, privilege := range privileges {
+			fmt.Fprintf(out, " - %s: %v\n", privilege.Name, privilege.Value)
```
Comment (tiborvass) [inline client/plugin_install.go]: @vdemeester did you fix this?

Comment (vdemeester) [inline client/plugin_install.go]: yes :angel:

Comment (tiborvass) [inline client/plugin_install.go]: thanks!

**Pull request merged**